.github/workflows/actions.yaml

```diff
@@ -11,6 +11,11 @@ jobs:
     steps:
       - name: Checkout the code
         uses: actions/checkout@v2
+      - name: Checkout gofsutil
+        uses: actions/checkout@v2
+        with:
+          repository: 'dell/gofsutil'
+          path: 'gofsutil'
       - name: Run the formatter, linter, and vetter
```

Comment:
Does this change mean gofsutils is being checked for linting/format/vetter as well?
If so, I don't think we want to do this- It could lead to our PRs bring blocked due to a change someone else made in a different repo entirely. 
Gofsutils/other repos have their own github actions in place. 

If only the driver is being checked- then we're fine and can ignore this comment

Reply:
@JacobGros if you view file you see 4 blocks , this seems to be ok

Reply:
> Does this change mean gofsutils is being checked for linting/format/vetter as well? If so, I don't think we want to do this- It could lead to our PRs bring blocked due to a change someone else made in a different repo entirely. Gofsutils/other repos have their own github actions in place.
> 
> If only the driver is being checked- then we're fine and can ignore this comment

Yeah, but I don't think there is a way to get around it? If we don't include gofsutil here, the checks fail because it's not there. I was concerned about failing our tests if linting fails in gofsutil, but they just had PR checks added to the repo, so the odds of that happening should be extremely low. So I don't think it's a huge concern?

Reply:
Hmm, could we exclude the gofsutil directory? If not, we can keep it as is.

Reply:
./... does not match packages in the vendor directory 
so checkout code you dont want to check into a folder named vendor